Tests/SentryTests/Integrations/Performance/Network/SentryNetworkTrackerTests.swift

```diff
@@ -653,19 +669,40 @@ class SentryNetworkTrackerTests: XCTestCase {
         XCTAssertEqual(sentryRequest.url, "https://www.domain.com/api")
         XCTAssertEqual(sentryRequest.method, "GET")
         XCTAssertEqual(sentryRequest.bodySize, 652)
-        XCTAssertEqual(sentryRequest.cookies, "myCookie")
-        XCTAssertEqual(sentryRequest.headers, headers)
+        XCTAssertNil(sentryRequest.cookies)
+        XCTAssertEqual(sentryRequest.headers, ["test": "test"])
         XCTAssertEqual(sentryRequest.fragment, "myFragment")
         XCTAssertEqual(sentryRequest.queryString, "query=myQuery")
     }
+
+    func testCaptureHTTPClientErrorRequest_noSecurityInfo() {
+        let sut = fixture.getSut()
+
+        let url = URL(string: "https://user:password@www.domain.com/api?query=myQuery#myFragment")!
+        var request = URLRequest(url: url)
+        request.allHTTPHeaderFields = fixture.securityHeader
+
+        let task = URLSessionDataTaskMock(request: request)
+        task.setResponse(createResponse(code: 500))
+        sut.urlSessionTask(task, setState: .completed)
+
+        guard let envelope = self.fixture.hub.capturedEventsWithScopes.first else {
+            XCTFail("Expected to capture 1 event")
+            return
+        }
+        let sentryRequest = envelope.event.request!
+
+        XCTAssertEqual(sentryRequest.url, "https://[Filtered]:[Filtered]@www.domain.com/api")
```

Comment:
Excellent, this tests that we use `removingPercentEncoding` in UrlDetail.sanitizedUrl 👍